shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/variantcallers/HaplotypeCallerGvcf.scala

```diff
@@ -42,18 +42,25 @@ class HaplotypeCallerGvcf(val parent: Configurable) extends Variantcaller {
   val haploidRegionsMale: Option[File] = config("haploid_regions_male")
   val haploidRegionsFemale: Option[File] = config("haploid_regions_female")
 
-  lazy val refernceSize = referenceDict.getReferenceLength
+  lazy val referenceSize = referenceDict.getReferenceLength
 
   lazy val fractionMale: Double = calculateFraction(haploidRegions, haploidRegionsMale)
   lazy val fractionFemale: Double = calculateFraction(haploidRegions, haploidRegionsFemale)
   lazy val fractionUnknown: Double = calculateFraction(haploidRegions, None)
 
+  /**
+   * This method will calculate the fraction of the given bed files of the used reference
+   *
+   * @param file1 Bed file
+   * @param file2 Bed file
+   * @return Fraction
+   */
   def calculateFraction(file1: Option[File], file2: Option[File]): Double = {
```

Comment:
If is not yet entirely clear to me why this has exactly two input files. A more generic case could be constructed as `calculateFraction(bedFiles: List[File])` using `BedRecordList.fromList`